sys/linux/dev_dri.txt

```diff
@@ -327,11 +326,10 @@ drm_mode_create_dumb {
 	height	int32
 	width	int32
 	bpp	int32
-	flags	int32
-# Outputs:
-	handle	drm_dumb_handle[opt]
-	pitch	const[0, int32]
-	size	const[0, int64]
+	flags	const[0, int32]
```

Comment:
This looks unrelated. Are you use it's always 0?

Reply:
> The flags field is currently unused and must be zeroed. Different flags to modify the behavior may be added in the future.

source: https://manpages.debian.org/testing/libdrm-dev/drm-gem.7.en.html

Reply:
If it's not used, it's good to declare it as `const[0, int32]`. Maybe clarify with a comment:
```
# Does not seem to be used by any driver.
flags	const[0, int32]
```

Reply:
[comment added](https://github.com/google/syzkaller/pull/2008/files#diff-26102468373666679c2ae06107ae0d6bR329)